### DISCUSSION TITLE
Rotate Data of 90degree around phi1

### DISCUSSION THREAD

**Discussion by SalomeParent** [mtex-toolbox/mtex#566] [Ask Anything]
Hello ! 

I want to rotate my data of 90degree around phi1 (Euler Angle) because my data are not well indexed.
Do you have any idea ?

My script is the following one: 

% crystal symmetry
CS = {... 
  'notIndexed',...
  crystalSymmetry('622', [2.989 2.989 13.615], 'X||a', 'Y||b*', 'Z||c', 'mineral', 'Ti2AlN', 'color', 'light blue')};

% plotting convention
setMTEXpref('xAxisDirection','east');
setMTEXpref('zAxisDirection','intoPlane');

%% Specify File Names
% path to files
pname = 'C:\Users\sparen01\Desktop\MATLAB';

% which files to be imported
fname = [pname '\2B_Y-1.ang'];

%% Import the Data
% create an EBSD variable containing the data
ebsd = loadEBSD(fname,CS,'interface','ACOM','convertEuler2SpatialReferenceFrame');

seg_angle = 2;
min_points = 10;
[grains,ebsd.grainId,ebsd.mis2mean] = calcGrains(ebsd('indexed'),'angle',seg_angle*degree);
grains = grains(grains.grainSize > min_points);
ebsd = ebsd(grains);
[grains,ebsd.grainId,ebsd.mis2mean] = calcGrains(ebsd('indexed'),'angle',seg_angle*degree)

plot(ebsd('ti'),ebsd('ti').orientations)
hold on
plot(grains.boundary)
hold off

%hold on
%text(grains,int2str(grains.id))
%hold off

%o_ref = orientation('Euler',[62,85,76]*degree, ebsd('Ti2AlN').CS) % some reference orientation, new syntax
%mori = o_ref .\ ebsd('Ti2AlN').orientations
%mangle = mori.angle/degree;

%plot(ebsd('Ti2AlN'),mangle)
%hold on
%plot(grains.boundary)
%hold off
%mtexColorbar

h=Miller (1,1,-2,1,ebsd('ti').CS)
plotPDF(ebsd(grains(6)).orientations,h,'projection','eangle','points','all','MarkerSize', 2,'MarkerFaceColor','r')
hold on
plotPDF(ebsd(grains(3)).orientations,h,'projection','eangle','points','all','MarkerSize', 2,'MarkerFaceColor','r','add2all')
hold off

Thank you for your answers.

Best regards

Salomé Parent

**Comment by ralfHielscher** [comment]
What about 
```matlab
ebsd = rotate(ebsd,90*degree)
```
Ralf.
